functions/Get-DbaSsisEnvironmentVariable.ps1

```diff
@@ -0,0 +1,281 @@
+<#
+.SYNOPSIS
+This command gets specified SSIS Environment and all its variables
+
+.DESCRIPTION
+This command gets all variables from specified environment from SSIS Catalog. All sensitive valus are decrypted.
+The function communicates directly with SSISDB database, "SQL Server Intergation Services" service isn't queried there.
+Each parameter (besides SqlInstance and SqlCredential) acts as the filter to only include or exclude particular element
+
+.PARAMETER SqlInstance
```

Comment:
from the parameter definition I see multiple sql instances are accepted after all.
Can you make that clear in the parameter help than, please?